File: PPD/app/src/main/java/com/yassou/postpartumapp/model/ChoicesDataModel.java
package com.yassou.postpartumapp.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class ChoicesDataModel implements Serializable {

    @SerializedName("title")
    private String title;

    @SerializedName("value")
    private Integer value;


    public ChoicesDataModel(String title, Integer value){
        this.title = title;
        this.value = value;
    }


    //title
    public String getTitle() {
        return title;
    }
    public void setTitle (String title){
        this.title = title;
    }



    //value
    public Integer getValue() {
        return value;
    }

    public void setValue(Integer value){
        this.value = value;
    }
}

File: PPD/app/src/main/java/app/yassou/postpartumapp/model/ChoicesDataModel.kt
package app.yassou.postpartumapp.model

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class ChoicesDataModel(
    @field:SerializedName("title")
    //title
    var title: String?, @field:SerializedName("value")
    //value
    var value: Int?) : Serializable